modules/assim.batch/R/pda.bayesian.tools.R

```diff
@@ -0,0 +1,187 @@
+##' Paramater Data Assimilation using BayesianTools R Package
+##'
+##'
+##' @title Paramater Data Assimilation using BayesianTools
+##' @param settings = a pecan settings list
+##'
+##' @return nothing. Diagnostic plots, MCMC samples, and posterior distributions
+##'  are saved as files and db records.
+##'
+##' @author Mike Dietze
+##' @author Istem Fer, Ryan Kelly
+##' @export
+pda.bayesian.tools <- function(settings, params.id=NULL, param.names=NULL, prior.id=NULL, chain=NULL, 
+                       iter=NULL, adapt=NULL, adj.min=NULL, ar.target=NULL, jvar=NULL, n.knot=NULL) {
+  
+  # library(devtools)
+  # install_url("https://dl.dropboxusercontent.com/s/hy9l6mokresqyel/BayesianTools_0.0.0.9000.tar.gz")
+  library(BayesianTools)
```

Comment:
That's arguable, but more pressing is that you'll need to add this to scripts/install.dependencies.R so that the package gets added to peoples systems.